### DISCUSSION TITLE
Use tilda for home directory

### DISCUSSION THREAD

**Discussion by leana8959** [sharkdp/fd#1424] [Q&A] [answered]
Hello !
I have a script that uses `fd` and `fzf`, the goal is to search through the file list to create a tmux session per project directory. I'm trying to improve it.

## The problem
The path names gets quite long, with `/Users/leana/` being repeated each time.
Is there a way that I could replace my `$HOME` path with a `~`? (i.e. in the screenshot below, truncate up to `repo/`.)
<img width="969" alt="image" src="https://github.com/sharkdp/fd/assets/87855546/8adcf605-78f9-47eb-ab22-f0dd9c5f5177">

This is (an excerpt of) the script that I use to generate the path I look for
```fish
set selected \
    (begin
        fd . $REPOS_PATH --exact-depth 2 --type d;
        fd . $PLAYGROUND_PATH --exact-depth 1 --type d;
    end 2> /dev/null | fzf)
```

## What I've tried
The closest to what I want is this:
`$ fd . --base-directory $REPOS_PATH --exact-depth 2 --type d` (from #1321)
This kinda works, but it also removes the `repos/` part. Since my script searches in multiple directories in `~`, this doesn't work unless I add part of the path striped away back, which is cumbersome.

Thanks for the project, and thanks for reading my question !

**Comment by sharkdp** [comment, marked as answer]
Could you pipe the `fd` results into `sed`? Something like
``` bash
sed -e "s@^$HOME/@~/@"
```

**Reply by leana8959** [reply]
Thank you, this is such an elegant and simple solution !

**Reply by xuhdev** [reply]
How can I adapt this solution to `fd --color=always`?

**Reply by lalvarezt** [reply]
> How can I adapt this solution to `fd --color=always`?

try:
`sed -e "s@$HOME/@~/@"`
just remove the start anchor
